apps/web/src/app/(dashboard)/documents/data-table-action-dropdown.tsx

```diff
@@ -174,6 +175,8 @@ export const DataTableActionDropdown = ({ row }: DataTableActionDropdownProps) =
         </DropdownMenuItem>
       </DropdownMenuContent>
 
+      <DocumentShareButton token={String(row.Recipient[0]?.token)} documentId={row.id} />
```

Comment:
This can just be `recipient.token` since we do a matching in the code above. No need to cast to string, we should update the share dialog to have token as an optional prop.